src/target/riscv/riscv-013.c

```diff
@@ -3114,20 +3132,19 @@ static int modify_privilege(struct target *target, uint64_t *mstatus, uint64_t *
 	return ERROR_OK;
 }
 
-static int read_memory_bus_v0(struct target *target, target_addr_t address,
-		uint32_t size, uint32_t count, uint8_t *buffer, uint32_t increment)
+static int read_memory_bus_v0(struct target *target, riscv_mem_access_info_t access)
```

Comment:
If you name the structure with arguments `args`, you will not need to rename the `access` variable in the function.

```suggestion
static int read_memory_bus_v0(struct target *target, riscv_mem_access_args_t args)
```

Side note, no change requested:

I am afraid that we do not have a good way of testing the functions for system bus v0 (similarly to other features like the 0.11 version of the debug spec or the BSCAN bridge).

Reply:
Structure renamed